rllib/evaluation/tests/test_env_runner_v2.py

```diff
@@ -0,0 +1,82 @@
+from random import sample
+import unittest
+
+import ray
+from ray.rllib.algorithms.ppo import PPO, PPOConfig
+from ray.rllib.examples.env.debug_counter_env import DebugCounterEnv
+from ray.rllib.examples.env.multi_agent import BasicMultiAgent
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.tune import register_env
+
+
+register_env("basic_multiagent", lambda _: BasicMultiAgent(2))
+
+
+class TestEnvRunnerV2(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        ray.init(num_cpus=1, local_mode=True)
```

Comment:
Don't use local_model and num_cpus once the test is finalized.

Reply:
oops, done